webapp/channels/src/components/suggestion/at_mention_provider/at_mention_provider.tsx

```diff
@@ -45,7 +45,6 @@ type Results = {
 type ResultsCallback = (results: Results) => void;
 
 export type Props = {
-    textboxId?: string;
```

Comment:
This reverts changes from #29900 which added IDs only to the suggestions given by the `AtMentionProvider`. Those aren't needed any more because the `SuggestionList` is now what adds the IDs instead of them